src/test/java/com/thebois/models/inventory/InventoryTests.java

```diff
@@ -179,96 +179,12 @@ public void inventoryDoesNotHaveMultipleOfItem() {
     public void addMultipleItemsToInventory() {
         // Arrange
         final Inventory inventory = new Inventory();
-        final ArrayList<IItem> items = new ArrayList<>();
-        items.add(new Log());
-        items.add(new Log());
 
         // Act
-        inventory.addMultiple(items);
+        inventory.addMultiple(List.of(new Log(), new Log()));
```

Comment:
Looks like you haven't merged from master in a while.

After my item refactor that you'll get from master, you'll have to use `ItemFactory.createItem(ItemType.LOG)` instead.